src/pagination/get_data_post.jsx

```diff
@@ -0,0 +1,238 @@
+    },
+    {
+        name: 'Design',
+        numberOf: '8'
+    },
+    {
+        name: 'Handmade',
+        numberOf: '7'
+    },
+    {
+        name: 'Interior',
+        numberOf: '1'
+    },
+    {
+        name: 'Wood',
+        numberOf: '6'
+    },
+]
+
+const recentPosts = [
+    {
+        path: 'notebook',
+        mainText: 'Going all-in with millennial design',
+        date: '03 Aug 2022'
+    },
+    {
+        path: 'exploring',
+        mainText: 'Exploring new ways of decorating',
+        date: '03 Aug 2022'
+    },
+    {
+        path: 'handmade',
+        mainText: 'Handmade pieces that took time to make',
+        date: '03 Aug 2022'
+    },
+    {
+        path: 'milan',
+        mainText: `Modern home in Milan`,
+        date: '03 Aug 2022'
+    },
+    {
+        path: 'redesign',
+        mainText: 'Colorful office redesign',
+        date: '03 Aug 2022'
+    },
+]
+
+const recentPostsList = recentPosts.map(post => {
+    return (
+        <div className='recent_post_item'>
+            <img width='80' height='80' src={`/photos/recent_posts/${post.path}.png`} alt={post.path} />
+            <div>
+                <p className='recent_post_text'>{post.mainText}</p>
+                <time className='recent_post_time' dateTime={post.date}>{post.date}</time>
+            </div>
+        </div>
+    )
+})
+
+const categoriesList = categories.map(category => {
+    return (
+        <div className='catergory_item'>
+            <button className='catergory_item_btn'>{category.name}</button>
+            <p className='catergory_item_text'>{category.numberOf}</p>
+        </div>
+    )
+})
+const PostInfoDisplayer = ({ photo, text }) => {
+    return (
+        <div className='post_info_item'>
+            <img width='20' height='20' src={`/photos/${photo}`} alt={text} />
+            <p className='post_info_item_text'>{text}</p>
+        </div>
+    )
+}
+
+const PostList = ({ posts }) => {
+    return posts.map((post) => {
+        return (
+            <div className='post_item'>
+                <img width='817' height='500' src={`/photos/${post.path}.png`} alt={post.path} />
+                <div className='post_info'>
+                    <PostInfoDisplayer photo={'user.svg'} text={post.person} />
+                    <PostInfoDisplayer photo={'calendar.svg'} text={post.date} />
+                    <PostInfoDisplayer photo={'ci_tag.svg'} text={post.type} />
+                </div>
+                <h2 className='post_title'>{post.title}</h2>
+                <p className='post_text'>{post.text}</p>
+                <div className='post_btn_container'><button className='post_btn'>Read more</button></div>
+            </div >
+        )
+    })
+
+
+}
+
+const PaginatedItemsPost = () => {
+    const [pageCount, setPageCount] = useState(0);
+    const [itemOffset, setItemOffset] = useState(0);
+    const [currentPage, setCurrentPage] = useState(JSON.parse(localStorage.getItem('localCurrentPage')) || 0);
```

Comment:
left similar comment in the other PR. pagination should not be saved in `localStorage` unless there is a very good reason for it. it should be synced with URL